R/as_wide_table.R

```diff
@@ -168,7 +166,7 @@ as_wide_table2 <- function(austraits){
     "latitude (deg)", "longitude (deg)", "location", "plot_id",
     
     #stuff relating to contexts and methods
-    "context", "methods", "method_id", "original_name",
+    "context", "methods", "original_name",
```

Comment:
So version 4.0 had `method_id` as a context category. For version 5.0, what had been v4.0 method_id became `method_context_id` and instead the name `method_id` was used in the methods table. I'm not quite sure if this matters here, but there is a `method_id` in both versions, just in different tables, with different meanings